resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -507,6 +512,12 @@ protected void doResolve(String inetHost,
             return;
         }
 
+        if (inetHost.isEmpty()) {
```

Comment:
we should move this check above `createByteArrayFromIpAddressString` which may throw a NPE if `null` is passed in.